### PR TITLE
libg15, libg15render, g15daemon and g15stats

### PR DESCRIPTION
**Pull request by Menelkir**
This is a group of packages necessary for Logitech G15 keyboards and other supported devices  (including G-keys and LCD Screen). I have a G15 keyboard for that matter and it works as expected.

#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me

### CODE REVIEW

**Comment by Chocimier** [comment]
This is your own software, do you use it on Void?

**Comment by Menelkir** [comment]
> 
> 
> This is your own software, do you use it on Void?

Yes, I've tested using void-glibc amd64. Actually isn't my software, but I've adopted and maintain it. I did all pertinent tests with my logitech g15 using it on void and it seems to work as expected (G buttons, display buttons, g15stats rotation, etc).  
I've tried my best reading the documentation so forgive me if the templates aren't that good.

File: srcpkgs/g15daemon/files/g15daemon/run [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh -e
+exec g15daemon
```
Comment (Chocimier) [inline srcpkgs/g15daemon/files/g15daemon/run:2]: Readme says that it needs to be root for logging, but logging by unprivileged user do work. Any problems with running as dedicated user crated as in [Manual.md](https://github.com/void-linux/void-packages/blob/master/Manual.md#creating-system-accountsgroups-at-runtime) and used as in [netdata](https://github.com/void-linux/void-packages/blob/189d87cab913dfca5bb5b33ea37c0965d4329264/srcpkgs/netdata/files/netdata/run)?

File: srcpkgs/g15stats/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'g15stats'
+pkgname=g15stats
+version=3.0.3
+revision=1
+build_style=gnu-configure
+CFLAGS=" -fcommon" # https://wiki.gentoo.org/wiki/Gcc_10_porting_notes/fno_common
+makedepends="libgtop-devel g15daemon pkg-config"
```
Comment (Chocimier) [inline srcpkgs/g15stats/template:7]: moving `pkg-config` to `hostmakedepends` fixes cross compilation.

**Review by Chocimier — COMMENTED** [review]
Good!
Please split headers, static libs and symlinks to dynamic libs to `*-devel` packages. (`xnew -a libg15 libg15-devel` will be good start)
Make one commit per template.

**Comment by Chocimier** [comment]
Please fix [xlint warnings](https://github.com/void-linux/void-packages/pull/30992/checks?check_run_id=2622798222) too.

**Pull request closed without merging**